### DISCUSSION TITLE
use public file with serverless api

### DISCUSSION THREAD

**Discussion by kukac7** [vercel/vercel#5357] [Help]
node build the html files whose contents i want to use with serverless api.
i tried the [`includeFiles`](https://vercel.com/knowledge/how-can-i-use-files-in-serverless-functions) setting, but to no avail.

```
{
	"functions": {
		"api/send.js": {
			"includeFiles": "public/*.html"
		}
	},
	"routes": [
		{
			"src": "/(.*)",
			"dest": "/$1.html"
		}
	]
}
```
npm build does this by creating the html files from the `mjml` folder to the `public` folder.

is there a solution for this?
thanks for the help!

**Comment by SawkaDev** [comment]
@kukac7  any luck with fixing?
